instance/migrations/0039_auto_20160416_1729.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0038_merge'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='openstackserver',
+            name='progress',
+        ),
+        migrations.RemoveField(
+            model_name='openstackserver',
+            name='status',
+        ),
+        migrations.AddField(
+            model_name='openstackserver',
+            name='_progress',
+            field=models.CharField(choices=[('failed', 'type'), ('running', 'type'), ('success', 'type')], default='running', db_column='progress', max_length=7),
+        ),
+        migrations.AddField(
+            model_name='openstackserver',
+            name='_status',
+            field=models.CharField(choices=[('active', 'type'), ('booted', 'type'), ('new', 'type'), ('provisioning', 'type'), ('ready', 'type'), ('rebooting', 'type'), ('started', 'type'), ('terminated', 'type')], default='new', db_column='status', max_length=20, db_index=True),
+        ),
+        migrations.AlterField(
+            model_name='generallogentry',
+            name='level',
+            field=models.CharField(choices=[('DEBUG', 'Debug'), ('INFO', 'Info'), ('WARNING', 'Warning'), ('ERROR', 'Error'), ('CRITICAL', 'Critical')], default='INFO', max_length=9, db_index=True),
+        ),
+        migrations.AlterField(
+            model_name='instancelogentry',
+            name='level',
+            field=models.CharField(choices=[('DEBUG', 'Debug'), ('INFO', 'Info'), ('WARNING', 'Warning'), ('ERROR', 'Error'), ('CRITICAL', 'Critical')], default='INFO', max_length=9, db_index=True),
+        ),
+        migrations.AlterField(
+            model_name='serverlogentry',
+            name='level',
+            field=models.CharField(choices=[('DEBUG', 'Debug'), ('INFO', 'Info'), ('WARNING', 'Warning'), ('ERROR', 'Error'), ('CRITICAL', 'Critical')], default='INFO', max_length=9, db_index=True),
+        ),
+    ]
```

Comment:
I find django migrations quite confusing... I guess this file is only here to be able to get the next migration to run and will disappear from this diff after https://github.com/open-craft/opencraft/pull/53 is merged and this PR rebased.